include/dpp/channel.h

```diff
@@ -91,6 +92,16 @@ enum overwrite_type : uint8_t {
 	ot_member = 1
 };
 
+/**
+ * @brief types to sort posts in a forum channel
+ */
+enum sort_order_type : uint8_t {
```

Comment:
@braindigitalis do you want `_t` or is `_type` fine?

Reply:
probably should be _t like the rest of these, but i know there are some that arent _t on them

Reply:
The rest of the ones in that header are `_type`